PyNite/Member3D.py

```diff
@@ -872,19 +872,32 @@ def shear_array(self, Direction, n_points: int, combo_name='Combo 1'):
             The number of points in the array to generate over the full length of the member.
         combo_name : string
             The name of the load combination to get the results for (not the load combination itself).
+        x_array : array = None
+            A custom array of x values that may be provided by the user, otherwise an array is generated.
+            Values must be provided in local member coordinates (between 0 and L) and be in ascending order
         """
         
-        # Segment the member into segments with mathematically continuous loads if not already done
```

Comment:
I'd like to have the documentation state why segmenting is necessary - to produce mathematically continuous beam segments (no point loads withing a segment). This allows us to use direct integration to obtain changes in moment, shear, slope and deflection inside the member. That may not be obvious to others.

Reply:
Agreed. If i'm honest, I can't remember why this was changed. I'll revert.